week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -13,9 +14,10 @@ function mood() {
 
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
+  let isBigEnough = num;
 
-  if (isBigEnough) {
+  // if (num => isBigEnough) {
+    if (isBigEnough => num) {
```

Comment:
Let's go through what is evaluated in this `if` condition now. You evaluate `isBigEnough => num`. However, earlier in the function you set `let isBigEnough = num`. So essentially, you evaluate `num => num`, which will always be true. So, let's say we have `let num = 1;` in the first line. What would we get as the result? Is it what you'd expect based on the function name?

Hint: Can you make `isBigEnough` to contain `true` when `num` is greater than `10` and false otherwise?

How would you now change the code so that you get the right result without changing `if` statement here?